### ISSUE TITLE
mutt missing dependecy?

### ISSUE THREAD

**Issue by henriqueleng** [voidlinux/void-packages#5763]
Hi,

I stayed more than an hour trying to set up mutt, and it was giving a 'no authenticators available' message.

I searched google and tried a bunch of different things until I found
mail on a mail list saying to install "cyrus-sasl-modules", and it solved the problem. Shouldn't this package be a dependency of mutt? Since it is needed by mutt?

Thanks,

Henrique N. Lengler

**Comment by henriqueleng** [comment]
Not only mutt, but mtsmtp and others.

**Comment by chneukirchen** [comment]
I guess there are many ways to use mutt that don't need this module, so making it a hard dependency is perhaps not a good idea.

**Comment by Duncaen** [comment]
A `INSTALL.msg` mentioning `cyrus-sasl-modules` and maybe other optional dependencies would be good i guess.
I think multiple users run into this problem by now.

**Comment by henriqueleng** [comment]
Yeah, just a way to tell users, so they don't waste a lot of time trying to figure out why tls isn't working.

**Comment by chneukirchen** [comment]
Just FTR, sasl is possibly needed for SMTP, but e.g. IMAP works fine without. This is unrelated to TLS.

**Comment by chneukirchen** [comment]
(Ok, some IMAP setups may need SASL too.)

**Comment by logenkain** [comment]
Yea, a message would be a good idea. It's funny how every time I do a new install I forget about it, fix it, and never think to add a message to the package...

**Comment by neton-bricio** [comment]
please, insert modifications in Neomutt
